service/handlers.go

```diff
@@ -1133,6 +1137,19 @@ func (h handler) createToken(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	tokens, err := h.dbClient.ListTokenEntries(ctx, projectName)
+	if err != nil {
+		level.Error(l).Log("message", "error listing tokens from DB", "error", err)
+		h.errorResponse(w, "error listing tokens", http.StatusInternalServerError)
+		return
+	}
+
+	if len(tokens) == numOfTokensLimit {
```

Comment:
I went back and forth deciding if the check should live here or in the CP. Ultimately decided here as the limitation shouldn't live within the CP implementation.

Reply:
I think here is what i would prefer also 👍🏽